docs/Projects/TAD/reporting_standard.md

```diff
@@ -0,0 +1,640 @@
+    5. `timestamp` (OPTIONAL). A timestamp of the date and time of the answer.
+
+## Example
+
+```yaml
+algorithm-registers:
+  - {id}                                                # Optional. Example: https://algoritmes.overheid.nl/nl/algoritme/45991543.
+language:
+  - {lang_0}                                            # Optional. Example nl.
+license: {licence}                                      # Required. Example: apache-2.0 or any license identifier from https://huggingface.co/docs/hub/repositories-licenses
+license_name: {licence_name}                            # Required. if license = other, specify an id for the licence. Example: 'my-license-1.0'
+license_link: {license_link}                            # Required. if license = other, specify "LICENSE" or "LICENSE.md" to link to a file of that name inside the repo, or a URL to a remote file.
+library_name: {library_name}                            # Optional. Example: keras or any library from https://github.com/huggingface/huggingface.js/blob/main/packages/tasks/src/model-libraries.ts.
+tags:
+- {tag_0}                                               # Optional. Example: audio
+- {tag_1}                                               # Optional. Example: automatic-speech-recognition
+upl: {upl_uri}                                          # Optional. Example: https://standaarden.overheid.nl/owms/terms/AangifteVertrekBuitenland
+owners:
+- organization: {organization_name}                     # Required. Example: BZK
+  oin: {oin}                                            # Optional. Example: 00000001003214345000
+  name: {owner_name}                                    # Optional. Example: John Doe
+  email: {owner_email}                                  # Optional. Example: johndoe@email.com
+  role: {owner_role}                                    # Optional. Example: Data Scientist.
+
+model-index:
+- name: {model_id}                                      # Required. Example: CatClassifier.
+  model: {model_uri}                                    # Required. URI to a repository containing the model file.
+  results:
+  - task:
+      type: {task_type}                                 # Required. Example: image-classification.
+      name: {task_name}                                 # Optional. Example: Image Classification.
+    dataset:
+      type: {dataset_type}                              # Required. Example: common_voice. Link to a repository containing the dataset or a dataset id from https://hf.co/datasets.
+      name: {dataset_name}                              # Required. Example: "Common Voice (French)". A pretty name for the dataset.
+      revision: {dataset_version}                       # Optional. Example: 5503434ddd753f426f4b38109466949a1217c2bb
+    metrics:
+    - type: {metric_type}                               # Required. Example: false-positive-rate. Use metric id from https://hf.co/metrics.
```

Comment:
First of all the terminology is a bit unfortunate, but that's because I stick to the terminology used by hugging face. They use "type" to denote their metric type, which is not a data type. I can add a field "dtype" to specify the datatype and add a suggestion that "name" can be the same as "type" which is the metric id.